flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/plan/stream/sql/join/IntervalJoinTest.scala

```diff
@@ -493,7 +493,7 @@ class IntervalJoinTest extends TableTestBase {
 
   @Test
   def testFallbackToRegularJoin(): Unit = {
-    // the following query would translated into regular join instead of interval join because the
+    // the following query would be translated into regular join instead of interval join because the
```

Comment:
We can remove the grammar correction.  @snuyanzin thoughts on changes like this in an unrelated PR?

Reply:
usually it could be done in same PR however in a separate hotfix commit
for example like here https://github.com/apache/flink/pull/24277